pages/common/lz4.md

```diff
@@ -0,0 +1,27 @@
+# lz4
+
+> Compress or decompress .lz4 files.
+
+- Compress a file:
+
+`lz4 {{file}}`
+
+- Decompress a file:
+
+`lz4 -d {{file.lz4}}`
+
+- Decompress a file and write to stdout:
+
+`lz4 -dc {{file.lz4}}`
+
+- Package and compress a directory and its contents:
+
+`tar cvf - {{path/to/dir}} | lz4 - {{dir.tar.lz4}}`
+
+- Decompress and unpack a directory and its contents:
+
+`lz4 -d {{dir.tar.lz4}} | tar -xv`
+
+- Compress a file using the best compression:
```

Comment:
This should have been the second example.

Reply:
Ah, oops! That might have been a good idea.

Most of the time, the best compression mode doesn't actually compress it much more than the standard compression does, whilst taking much longer. So to that end it's not actually that useful in day-to-day circumstances.

Reply:
I thought so, too at first. I decided against it as for me LZ4's major benefit is its default mode (`-1`): the space savings on compressable files are nicely balanced with the speed of processing on incompressable files. The improved compression of using `-9` are diminishing the benefits of processing speed, and as tldr is making an effort to appeal to novice users, I thought I'd intentionally discourage the use of `-9` by putting it at the end.

Reply:
Ok, fair enough.